pkg/util/trace.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"errors"
+
+	"contrib.go.opencensus.io/exporter/ocagent"
+	"go.opencensus.io/trace"
+)
+
+// InitializeTracing initiates trace, exporter and the sampler
+func InitializeTracing(serviceName string, address string) error {
+	if address != "" {
+		oce, err := ocagent.NewExporter(
+			ocagent.WithInsecure(),
+			ocagent.WithAddress(address),
+			ocagent.WithServiceName(serviceName))
+		if err != nil {
+			return err
+		}
+		trace.RegisterExporter(oce)
+		trace.ApplyConfig(trace.Config{
+			DefaultSampler: trace.AlwaysSample(),
+		})
+		return nil
+	}
+	return errors.New("collector address is empty")
```

Comment:
Cross-referencing #3539, doesn't `address == ""` imply that tracing is disabled?

Reply:
Can we just not call this function in all the `main`s if `*traceCollector` is empty?